==== triton/lib/Target/LLVMIR/LLVMDIScope.cpp#8 - /google/src/cloud/wcui/mlir_6f6336858e4588ebd113ebcc930f6384a4edca54_1712678792/triton/lib/Target/LLVMIR/LLVMDIScope.cpp ====
# action=edit type=text
--- triton/lib/Target/LLVMIR/LLVMDIScope.cpp	2024-02-09 08:45:23.000000000 -0800
+++ triton/lib/Target/LLVMIR/LLVMDIScope.cpp	2024-04-09 09:10:04.000000000 -0700
@@ -90,7 +90,8 @@
         compileUnitAttr = LLVM::DICompileUnitAttr::get(
             context, distinctId, llvm::dwarf::DW_LANG_C, fileAttr,
             StringAttr::get(context, "triton"),
-            /*isOptimized=*/true, LLVM::DIEmissionKind::LineTablesOnly);
+            /*isOptimized=*/true, LLVM::DIEmissionKind::LineTablesOnly,
+            LLVM::DINameTableKind::Default);
       }
       subprogramFlags = subprogramFlags | LLVM::DISubprogramFlags::Definition;
     } else {
